File: All Lab Tasks/Lab 1/RideSharingApp__Strategy_Pattern_/src/DigitalWalletPayment.java
public class DigitalWalletPayment implements IPaymentMethod
{
    public String WalletId;
    public int Otp;
    public DigitalWalletPayment(String walletId, int otp)
    {
        WalletId = walletId;
        Otp = otp;
    }

    public void ProcessPayment(double amount)
    {

            System.out.println("Processing Digital Wallet payment of " + amount);
            System.out.println("Wallet ID: " + WalletId + ", OTP: " + Otp);

    }


}

File: Lab_1_RideSharingApp/Lab_1_RideSharingApp/Payment/DigitalWalletPayment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab_1_RideSharingApp
{
    public class DigitalWalletPayment : IPaymentMethod
    {
        public string WalletId { get; private set; }
        public int Otp { get; private set; }

        // Constructor
        public DigitalWalletPayment(string walletId, int otp)
        {
            WalletId = walletId;
            Otp = otp;
        }

        // Implementing IPaymentMethod interface
        public void ProcessPayment(double amount)
        {
            Console.WriteLine($"Processing Digital Wallet payment of {amount}.");
            Console.WriteLine($"Wallet ID: {WalletId}, OTP: {Otp}");
        }
    }
}
